docs/rackhd/tftp_dhcp_server.rst

```diff
@@ -0,0 +1,313 @@
+TFTP and DHCP Service Setup
+---------------------------
+.. _dhcpd.conf: http://linux.die.net/man/5/dhcpd.conf
+.. _ISC DHCP Server: https://www.isc.org/downloads/dhcp
+.. _on-tftp: https://github.com/RackHD/on-tftp
+.. _on-dhcp-proxy: https://github.com/RackHD/on-dhcp-proxy
+.. _on-imagebuilder: https://github.com/RackHD/on-imagebuilder
+
+.. _config.json: https://github.com/RackHD/RackHD/blob/master/packer/ansible/roles/monorail/files/config.json
+.. _RackHD iPXE: https://bintray.com/rackhd/binary/on-imagebuilder#files/ipxe
+
+RackHD is flexiable to adapt to different network environments for TFTP and DHCP service.
+By default, RackHD use `on-tftp`_ for TFTP service, `ISC DHCP Server`_ and DHCP proxy `on-dhcp-proxy`_ for DHCP service,
+and they are deployed in RackHD server along with other RackHD service `on-http`, `on-taskgraph`, `on-syslog`.
+They could be replaced with other TFTP and DHCP services, and also could be deployed to another separated server.
+
++---------------------------+---------------------------------------------+-------------------------------------------------+
+| Cases                     |            Supported TFTP Service           |             Supported DHCP Service              |
++===========================+=============================================+=================================================+
+| TFTP and DHCP service are | 1. on-tftp(default)                         | 1. ISC DHCP + on-dhcp-proxy(default)            |
+| in RackHD server          | 2. Third-party TFTP service such as         | 2. ISC DHCP only                                |
+|                           |    in.tftpd(tftp-hpa) in Ubuntu OS          | 3. Third-party DHCP Service + DHCP proxy        |
+|                           |                                             | 4. Third-party DHCP Service only                |
++---------------------------+---------------------------------------------+-------------------------------------------------+
+| TFTP and DHCP service are | 1. on-tftp                                  | 1. ISC DHCP + on-dhcp-proxy                     |
+| in separate server        | 2. Third-party TFTP service such as         | 2. ISC DHCP only                                |
+|                           |    in.tftpd(tftp-hpa) in Ubuntu OS          | 3. Third-party DHCP Service + DHCP proxy        |
+|                           |                                             | 4. Third-party DHCP Service only                |
++---------------------------+---------------------------------------------+-------------------------------------------------+
+
+**NOTE:** "Third-party" service means it's not RackHD default service.
+
+TFTP and DHCP are in RackHD Server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+TFTP Service Configuration in RackHD Server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The three section headings in short order "look" a little odd in the rendered view - not sure we need all three here, could we reduce it to two layers and still keeping the organizational structure you're after here?

For example, this is what I see in a render:
![screen shot 2016-08-25 at 9 21 55 am](https://cloud.githubusercontent.com/assets/43388/17977163/6bb023c8-6aa5-11e6-99c4-e6de0b0e6395.png)

and the table of contents:
![screen shot 2016-08-25 at 9 22 29 am](https://cloud.githubusercontent.com/assets/43388/17977181/764d9798-6aa5-11e6-8f94-a7abe090b3db.png)